sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md

```diff
@@ -36,6 +36,7 @@ Set-SPOSite [-Identity] <SpoSitePipeBind> [-AllowSelfServiceUpgrade <Boolean>] [
  [-ShowPeoplePickerSuggestionsForGuestUsers <Boolean>] [-StorageQuotaReset]
  [-DefaultSharingLinkType] [-DefaultLinkPermission] [-DefaultLinkToExistingAccess]
  [-ConditionalAccessPolicy <SPOConditionalAccessPolicyType>] [-LimitedAccessFileType <SPOLimitedAccessFileType>] [-AllowEditing <Boolean>] [-AnonymousLinkExpirationInDays <Int32>] [-OverrideTenantAnonymousLinkExpirationPolicy <Boolean>] [-SensitivityLabel <String>][<CommonParameters>]
+ [-RemoveLabel] [<CommonParameters>]
```

Comment:
According to PlatyPs, this parameter belongs to a new Parameter Set named ParamSet5, please see and update, thanks

### ParamSet5
```
Set-SPOSite [-Identity] <SpoSitePipeBind> [-RemoveLabel] [-WhatIf] [-Confirm] [<CommonParameters>]
```